lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_StatePMPSEnables.TcPOU

```diff
@@ -98,13 +103,18 @@ mc_power(
 IF nGoalStateIndex > 0 AND nGoalStateIndex <= GeneralConstants.MAX_STATES THEN
     IF astPositionState[nGoalStateIndex].bValid AND astPositionState[nGoalStateIndex].bUpdated THEN
         bValidGoal := TRUE;
+        bLockBounds := TRUE;
```

Comment:
This means that we stay locked to the most recent setpoint at all times, right? I think it's a good implementation, solid and not overcomplicated.